pkg/module/sync/runner_impl.go

```diff
@@ -138,9 +137,11 @@ func (r *RunnerImpl) setupModule(module *common.Module, kyma *v1beta2.Kyma) erro
 	return nil
 }
 
-func (r *RunnerImpl) SyncModuleStatus(ctx context.Context, kyma *v1beta2.Kyma, modules common.Modules) {
+func (r *RunnerImpl) SyncModuleStatus(ctx context.Context, kyma *v1beta2.Kyma, modules common.Modules,
+	removeModuleStateMetrics func(kymaName, moduleName string),
```

Comment:
Thing is, my first idea was to pass a function here. But then, since we finally strife to build up our unit test coverage now, how do you assert that in case of `moduleStatus.Manifest == nil` in `DeleteNoLongerExistingModuleStatus` that this function has been called? Is it possible to verify that?